app/lib/services/voip/useCallStore.test.ts

```
@@ -38,16 +38,20 @@ function createMockCall(callId: string) {
-		muted: false,
-		held: false,
-		remoteMute: false,
-		remoteHeld: false,
+		localParticipant: {
+			muted: false,
+			held: false,
+			contact: { id: 'u', displayName: 'U', username: 'u', sipExtension: '' },
+			setMuted: jest.fn(),
+			setHeld: jest.fn()
+		},
+		remoteParticipants: [{
```

ESLint

[error] Insert `⏎↹↹↹`

(prettier/prettier)